### ISSUE TITLE
Move to Flexbox, Plugins

### ISSUE THREAD

**Issue by KingPixil** [KingPixil/wing#28]
Wing will be moving to flexbox, give your opinion here.
#### Reasons for moving to flexbox
- Cleaner code
- Less file size (more features can be added now)
- 97% of browsers support flexbox

Now, utility classes like `.center` and `.vertical-align` are easily implemented. The grid will be given a revamp, with simpler classes and shortcuts. 

You will be able to do

``` html
<div class="row">
    <div class="col">Automatically Sized</div>
    <div class="col">Automatically Sized</div>
</div>
```

While still having compatibility for `col-1` through `col-12`, so you don't have to edit all of your existing code.

There will be grid offsets, this is a very much requested feature.

Other things will be added in a plugin system, where extra modules such as:
- Navbar
- Alert messages
- Full screen hero

Let me know what you think, and if you have any ideas, comment them below as well, this will all happen in Wing `v0.2.0`. And all of these changes will lead up to `v1.0.0`.

**Comment by CodeBrauer** [comment]
Nice! Looking forward for this features!

**Comment by KingPixil** [comment]
Thanks @CodeBrauer 👍 

I'll try to add as many as I can, the 1.0 development is in the v1.0.0 branch if you'd like to contribute.

**Comment by KingPixil** [comment]
Some more updates:

- Wing will be using system fonts, which means it won't need to import Open Sans anymore
- Components planned are:
  - Navigation
  -

**Comment by CodeBrauer** [comment]
I think 5kb is more than ok. It's not even a second on small mobile devices.

**Comment by KingPixil** [comment]
Alright, it's been merged in #45

**Comment by Globegitter** [comment]
@kbrsh have you ever release the plugin for a Full screen hero? Can not find any reference to it but would need this functionality.
